docs/source/guide/tutorials/base/how_to_train/instance_segmentation.rst

```diff
@@ -1,2 +1,327 @@
+
+``otx eval`` will output a F-measure for instance segmentation.
+
+2. The command below will run validation on our dataset
+and save performance results in ``performance.json`` file:
+
+.. code-block::
+
+  (otx) ...$ otx eval --test-data-roots /home/<username>/training_extensions/tests/assets/car_tree_bug \
+                      --load-weights models/weights.pth \
+                      --save-performance performance.json
+
+We will get a similar to this validation output:
+
+.. code-block::
+
+  ...
+
+  2023-02-21 22:37:10,263 | INFO : Inference completed
+  2023-02-21 22:37:10,263 | INFO : called evaluate()
+  2023-02-21 22:37:10,265 | INFO : F-measure after evaluation: 0.33333333333333326
+  2023-02-21 22:37:10,265 | INFO : Evaluation completed
+  Performance(score: 0.33333333333333326, dashboard: (1 metric groups))
+
+.. note::
+
+  You can omit ``--test-data-roots`` if you are currently inside a workspace and have test-data stuff written in ``data.yaml``.
+
+  Also, if you're inside a workspace and ``weights.pth`` exists in ``models`` dir, you can omit ``--load-weights`` as well, assuming those weights are the default as ``models/weights.pth``.
+
+  If you omit ``--save-performance``, it will create a ``performance.json`` in the folder for those weights.
+
+
+The output of ``./outputs/performance.json`` consists of dict with target metric name and its value.
+
+.. code-block::
+
+  {"f-measure": 0.33333333333333326}
+
+*********
+Export
+*********
+
+1. ``otx export`` exports a trained Pytorch `.pth` model to the 
+OpenVINO™ Intermediate Representation (IR) format.
+
+It allows running the model on the Intel hardware much more efficient, especially on the CPU. Also, the resulting IR model is required to run POT optimization. IR model consists of 2 files: ``openvino.xml`` for weights and ``openvino.bin`` for architecture.
+
+2. We can run the below command line to export the trained model
+and save the exported model to the ``openvino_model`` folder.
+
+.. code-block::
+
+  (otx) ...$ otx export --load-weights models/weights.pth \
+                        --save-model-to openvino_model
+
+  ...
+  [ SUCCESS ] Generated IR version 11 model.
+  [ SUCCESS ] XML file: /tmp/OTX-task-51omlxb0/stage00_DetectionExporter-train/model.xml
+  [ SUCCESS ] BIN file: /tmp/OTX-task-51omlxb0/stage00_DetectionExporter-train/model.bin
+
+  2023-02-21 22:38:21,893 - mmdeploy - INFO - Successfully exported OpenVINO model: /tmp/OTX-task-51omlxb0/stage00_DetectionExporter-train/model_ready.xml
+  2023-02-21 22:38:21,894 | INFO : run task done.
+  2023-02-21 22:38:21,940 | INFO : Exporting completed
+
+3. We can check the accuracy of the IR model and the consistency between the exported model and the PyTorch model,
+using ``otx eval`` and passing the IR model path to the ``--load-weights`` parameter.
+
+.. code-block::
+
+  (otx) ...$ otx eval --test-data-roots /home/<username>/training_extensions/tests/assets/car_tree_bug \
```

Comment:
Also, in all commands we can use just "workspace/splitted_dataset/ .... " and not mention the "/home/<username>"

Reply:
Yes, I'll fix it.